spyder_kernels/customize/spyderpdb.py

```diff
@@ -743,11 +754,13 @@ def set_spyder_breakpoints(self, breakpoints):
                     logger.debug(
                         "Could not send a Pdb continue call to the frontend.")
 
-    def notify_spyder(self):
-        """Send kernel state to the frontend."""
+    def publish_pdb_state(self):
+        """Send debugger state (frame position) to the frontend. The state is
+        only sent if it has changed since the last update."""
```

Comment:
```suggestion
        """
        Send debugger state (frame position) to the frontend.

        The state is only sent if it has changed since the last update.
        """
```

Improve formatting of this docstring.